modules/openapi-generator/src/main/resources/aspnetcore/2.1/Dockerfile.mustache

```diff
@@ -0,0 +1,18 @@
+FROM microsoft/aspnetcore-build:2.0 AS build-env
```

Comment:
@SeanFarrow do we need to update the line 1 and 15 to use `microsoft/aspnetcore-build:2.1` instead?